### ISSUE TITLE
Transparency

### ISSUE THREAD

**Issue by ktaylora** [anomal/RainbowVis-JS#15]
How hard would it be to add support for transparency via an alpha channel, such as : #00FFFFFF?

**Comment by anomal** [comment]
Are alpha channels represented like this in some context? (I am normally a back end developer, so I have not seen alpha channels appended like this before.)

**Comment by NicoWeio** [comment]
@anomal The alpha value is usually added to the end of the hex string. `#00FFFFFF` therefore represents a turquoise with 100% opacity.

**Comment by ExplodingCabbage** [comment]
> Are alpha channels represented like this in some context?

Yes; it's how hex colors work in CSS. See the "#rrggbbaa" section at https://developer.mozilla.org/en-US/docs/Web/HTML/Applying_color#RGB_values.

**Comment by NicoWeio** [comment]
So would somebody be able to implement that?
